magnet_coils.py

```diff
@@ -538,7 +538,7 @@ def magnet_coils(magnets, tor_ext, export_dir, logger = None):
         vol_ids = cut_mags(tor_ext, vol_ids, r_avg)
     
     # Export magnet coils
-    export_path = Path(export_dir) / Path(magnets["name"]).with_suffix('step')
+    export_path = Path(export_dir) / Path(magnets["name"]).with_suffix('.step')
```

Comment:
I think this '.' was removed in another PR somewhere

Reply:
without the '.' python complains about an invalid suffix, adding the '.' results in the magnets step file being exported, at least for me

Reply:
I must have misremembered that change - maybe it was making this same fix as this